src/pages/dashboardTab/store/dashboardReducer.js

```diff
@@ -725,20 +725,23 @@ const reducers = {
       } : { ...state };
   },
   SORT_SINGLE_GROUP_CHECKBOX: (state, item) => {
-    const groupData = state.checkbox.data.filter((d) => item.groupName === d.groupName)[0];
+    const groups = state.checkbox.data;
```

Comment:
One small change, we don't need a separate variable    "groups" as we are not reusing it. I know it's for readability but the "group" iterator in the filter can provide enough.

Reply:
Sure. Updating.

Reply:
Updated !